vividus-tests/src/main/resources/story/integration/PageStepsTests.story

```diff
@@ -31,12 +31,14 @@ Examples:
 |PAGE_LOAD_TIME       |
 
 Scenario: Verify URL components of the opened page
-Given I am on page with URL `${vividus-test-site-url}/mouseEvents.html`
+Given I am on page with URL `${vividus-test-site-url}/mouseEvents.html?query=%E2%82%AC`
```

Comment:
do we have URL query handling at this page?

Reply:
I don't think so, It loads the same page without any changes. Can be removed if necessary